server/review.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-github/v33/github"
+	"github.com/sourcegraph/go-diff/diff"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+const mlogReviewCommentBody = "Gentle reminder to check our logging [principles](https://docs.google.com/document/d/160-9Qfk-1a2_gOU-VHfZ5CbUdkY8BZxq-2VL7XYwqGA) before merging this change."
```

Comment:
I will update this after the https://mattermost.atlassian.net/browse/MM-32010 Also I'm open to better sentences 😅

Reply:
Yeah, we should add the contents of the document to this section: https://github.com/mattermost/mattermost-developer-documentation/pull/751. Shall we wait until the PR is merged?

Reply:
Deal 🤝

Reply:
Actually, that PR will take some time. We can go ahead with this pointing to the google doc for now if you prefer.